mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -210,6 +210,8 @@
     // not used in CB, but used in CDP CLI, so we need this in mock for local development
     private static final String CDP_CP_CUSTOM_DL_TEMPLATE = "CDP_CM_ADMIN_CREDENTIALS";
 
+    private static final String CDP_AZURE_ENABLE_DISK_SSE_WITH_CMK = "CDP_AZURE_ENABLE_DISK_SSE_WITH_CMK";
```

Comment:
Please remove and use the `Entitlement` constant static import instead. `CDP_CP_CUSTOM_DL_TEMPLATE` needs to be given as a string constant because it does not appear in `Entitlement`, yet it has to be added to the `getAccount()` output for sake of CDP CLI.